jellyfin.ruleset

```diff
@@ -5,6 +5,8 @@
     <Rule Id="SA1202" Action="Info" />
     <!-- disable warning SA1204: Static members must appear before non-static members -->
     <Rule Id="SA1204" Action="Info" />
+    <!-- disable warning SA1404: Code analysis suppression should have justification -->
+    <Rule Id="SA1404" Action="Info" />
```

Comment:
Why disable this? I did a quick check and it looks like there's only three places where we're using `[SuppressMessage]` without a justification, wouldn't it be better to just add a justification message to those three?